packages/apps/transfer/locales/en/common.json

```diff
@@ -99,5 +99,13 @@
   "Transfer pending - wating for proof": "Transfer pending - wating for proof",
   "Transfer pending - waiting for continuation command": "Transfer pending - waiting for continuation command",
   "Could not obtain proof information": "Could not obtain proof information",
-  "Transfer completed successfully": "Transfer completed successfully"
+  "Transfer completed successfully": "Transfer completed successfully",
+  "form-status-title-erroneous": "Something went wrong",
```

Comment:
I think we need to decide what kind of keys we should use here. Regular text, or domain-specific, or something else

Reply:
True, I don't really mind which one, this approach seemed more logical to me. I guess I will transform it to be in line with the approach you took

Reply:
Will be refactored later, in a separate PR. Created a ticket for this; https://app.asana.com/0/1204958723576385/1205129264294989/f